src/sentry/runner/commands/run.py

```diff
@@ -412,10 +412,12 @@ def inner(f):
 @run.command("ingest-consumer")
 @log_options()
 @click.option(
+    "consumer_types",
     "--consumer-type",
-    default=None,
+    default=["events", "transactions", "attachments"],
```

Comment:
Can we rather move the default into the `devserver` command? That way the caller has to be explicit.  Also, would it make sense to add an explicit `"all"` variant?

Reply:
I added a `--all-consumer-types` option